public/assets/js/scripts.js

```diff
@@ -2,9 +2,22 @@ function toggleQuestions(id) {
   $('#payment, #profile, #security, #getting-started').hide();
   $("#"+id).show();
 }
+function validateContactUsForm(){
+  if (
```

Comment:
This validation doesn't seem necessary since you are using the default HTML 5 validation. We will need to consider a better way to validate the form. It would have been better to use bootstrap validation but that can be done in another PR.